imod/mf6/package.py

```diff
@@ -278,6 +279,17 @@ def _validate(self, schemata: Dict, **kwargs) -> Dict[str, List[ValidationError]
                         errors[variable].append(e)
         return errors
 
+    def is_empty(self) -> bool:
+        """
+        Returns True if the package is empty- for example if it contains only no-data values.
+        """
+        allnodata_schemata = filter_schemata_dict(
```

Comment:
I think this comment is still useful:

```python
        # Create schemata dict only containing the
        # variables with a AllNoDataSchema and EmptyIndexesSchema (in case of
        # HFB) in the write schemata.
```

Reply:
done